letsencrypt/tests/auth_handler_test.py

```diff
@@ -22,9 +22,11 @@ class ChallengeFactoryTest(unittest.TestCase):
     def setUp(self):
         from letsencrypt.auth_handler import AuthHandler
 
-        # Account is mocked...
+        # Account and config are mocked
+        self.config = mock.MagicMock(
+            tls_sni_01_port=1234, http01_port=4321)
```

Comment:
`self.config` is not used anywhere in functions tested in this class, right? why not `self.config = mock.Mock()` then?